templates/design_system/include/head.html

```diff
@@ -30,11 +30,11 @@
           rel="stylesheet">
 
     <!-- Bootstrap CSS -->
-    <link rel="stylesheet" href="{% static 'bootstrap/css/bootstrap.min.css' %}" type="text/css"text/css">
+    <link rel="stylesheet" href="{% static 'cms/bootstrap/css/bootstrap.min.css' %}" type="text/css"text/css">
```

Comment:
Might we instead want to delete the cms folder here, and move bootstrap up a level? Our folder structure in the sitestatic folder seems a little unnecessarily complicated.

Reply:
@ekivemark Agree with @samgensburg-gov I thought I had made this change already. the design_system is a separate folder.